pages/common/odps-func.md

```diff
@@ -0,0 +1,19 @@
+# odps func
+
+> Manage functions in ODPS (Open Data Processing Service).
+
+- Show functions in the current project:
+
+`list functions;`
+
+- Create a Java function using a .jar resource:
+
+`create function {{func_name}} as {{path.to.package.Func}} using '{{package.jar}}';`
+
+- Create a Python function using a .py resource:
+
+`create function {{func_name}} as {{script.Func}} using 'script.py';`
```

Comment:
Sorry I missed this before, but shouldn't `script.py` be wrapped in `{{}}` just like `package.jar` above?